smithy-kotlin-codegen/src/main/kotlin/software/amazon/smithy/kotlin/codegen/core/RuntimeTypes.kt

```diff
@@ -48,6 +48,10 @@ object RuntimeTypes {
             val HttpResponse = runtimeSymbol("HttpResponse", KotlinDependency.HTTP, "response")
         }
 
+        object Retries {
+            val StandardRetryFeature = runtimeSymbol("StandardRetryFeature", KotlinDependency.HTTP, "retries")
```

Comment:
nit

I should have caught this before but this probably should have lived in `aws.smithy.kotlin.runtime.http.middleware` which is an existing package. Also since it takes a strategy and a policy nothing makes it "Standard". It could probably just be `RetryMiddleware`

Reply:
Well you couldn't have caught this before since it's new in this PR. 😃 I can relocate it.